Core/HLE/sceDisplay.cpp

```diff
@@ -764,11 +763,7 @@ void __DisplayFlip(int cyclesLate) {
 
 	// postEffectRequiresFlip is not compatible with frameskip unthrottling, see #12325.
 	if (g_Config.iRenderingMode != FB_NON_BUFFERED_MODE && !(unthrottleNeedsSkip && !FrameTimingThrottled())) {
-		if (shaderInfo) {
-			postEffectRequiresFlip = (shaderInfo->requires60fps || duplicateFrames);
-		} else {
-			postEffectRequiresFlip = duplicateFrames;
-		}
+		postEffectRequiresFlip = duplicateFrames || PostShaderChainRequires60FPS(GetFullPostShadersChain(g_Config.vPostShaderNames));
```

Comment:
Computing the 60fps requirement here is not ideal. since it's only once per frame though, I'm not sure it's worth caching...

Reply:
Let's cache it I guess, I see no harm in it anyway, at worst a negligible gain :)